src/log4net.sln

```diff
@@ -1,28 +1,10 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2012
```

Comment:
don't forget to add this license back in again, otherwise someone will just ask you to later 🤣

Reply:
check